platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -2047,7 +2047,10 @@ public OnInfoWindowCloseListener getOnInfoWindowCloseListener() {
   //
 
   /**
-   * Takes a snapshot of the map.
+   * Renders a snapshot of the current map state as a {@link Bitmap}.
```

Comment:
- we tend to not use any `{@link .. }` in the first line of the javadoc, this is to avoid generating links directly in overview pages. 
 - we tend to not link against Android SDK classes (scope for linking is only within the SDK)